src/RtcDS3231.cpp

```diff
@@ -136,7 +136,7 @@ RtcDateTime RtcDS3231::GetDateTime()
     uint8_t month = BcdToUint8(monthRaw & 0x7f);
     
 
-    return RtcDateTime(year, month, dayOfMonth, hour, minute, second);
+    return RtcDateTime(year, month, dayOfMonth, deyOfWeek, hour, minute, second);
```

Comment:
did you compile before creating the pull?
spelling problem here, deyOfWeek; this should be dayOfWeek.